frontend/src/components/create-contract/PaymentSummary.tsx

```diff
@@ -1,11 +1,29 @@
-const PaymentSummary = () => {
+import { CreateFormValues } from 'components/forms/types';
+
+//
+const PaymentSummary = ({
+  tokenName,
+  data,
+}: {
+  tokenName?: string;
+  data: CreateFormValues;
+}) => {
+  const token = tokenName || process.env.REACT_APP_PAYMENT_TOKEN_NAME || '';
+
+  const bounty = data.bounty ? parseInt(data.bounty) : 0;
+  const buyIn = bounty * 0.1;
```

Comment:
Assuming this is for testing, but we could just use the form value now right?

Reply:
I thought the value was actually hard-coded at 10%.